<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KIPMsgEncodingConfigDialogBase</class>
 <widget class="QDialog" name="KIPMsgEncodingConfigDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encoding Configuration</string>
  </property>
  <widget class="KPushButton" name="m_ApplyButton" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string>&amp;Apply</string>
   </property>
   <property name="shortcut" stdset="0">
    <string>Alt+A</string>
   </property>
  </widget>
  <widget class="KPushButton" name="m_OKButton" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut" stdset="0">
    <string>Alt+O</string>
   </property>
  </widget>
  <widget class="KPushButton" name="m_CancelButton" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>410</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string>Ca&amp;ncel</string>
   </property>
   <property name="shortcut" stdset="0">
    <string>Alt+N</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>430</width>
     <height>120</height>
    </rect>
   </property>
   <property name="title">
    <string>Entried host information</string>
   </property>
   <widget class="KComboBox" name="m_EncodingCombobox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="currentItem" stdset="0">
     <number>8</number>
    </property>
    <item>
     <property name="text">
      <string>Big5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Big5-HKSCS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>eucJP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>eucKR</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GB2312</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GBK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GB18030</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>JIS7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shift-JIS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TSCII</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>utf8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KOI8-R</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>KOI8-U</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-8-i</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ISO8859-15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IBM 850</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>IBM 866</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP874</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1251</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1252</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1253</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1254</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1255</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1256</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1257</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CP1258</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Apple Roman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TIS-620</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="m_GroupNameLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>230</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="m_UserNameLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>230</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nickname</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>GroupName</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Encoding</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>430</width>
     <height>260</height>
    </rect>
   </property>
   <property name="title">
    <string>Entried host</string>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <widget class="QListView" name="m_HostListView">
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kcombobox.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_HostListView</sender>
   <signal>currentChanged(QModelIndex &amp;, QModelIndex &amp;)</signal>
   <receiver>KIPMsgEncodingConfigDialogBase</receiver>
   <slot>slotCurrentHostChanged(QModelIndex &amp;, QModelIndex &amp;)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_OKButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgEncodingConfigDialogBase</receiver>
   <slot>slotOkClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ApplyButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgEncodingConfigDialogBase</receiver>
   <slot>slotApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgEncodingConfigDialogBase</receiver>
   <slot>slotCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_EncodingCombobox</sender>
   <signal>activated(int)</signal>
   <receiver>KIPMsgEncodingConfigDialogBase</receiver>
   <slot>slotEncodingComboChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
